docs/pages/es/installation/windows.md

```diff
@@ -83,7 +83,7 @@ wsl --set-default-version 2
 <b>Advertencia:</b>
 Se recomienda actualizar a <a href="#wsl2">WSL2</a> ya que WSL1 no soporta la aplicación Docker Desktop para Windows. En su lugar, solo soporta la versión obsoleta, [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/).
 
-Si aun desea continuar, siga las instrucciones para <button onclick="HideToggleFunction()"><b>WSL1</b></button>
+Si aun desea continuar, siga las instrucciones para <button class="WSL1-button" onclick="HideToggleFunction()"><b>WSL1</b></button>
```

Comment:
Good work, @Shulammite-Aso. You accounted for the Spanish version, too. 😄

Reply:
Yeah @leecalcote :laughing:  These were the two places i saw the button in use.